docs/experiments-microblog.md

```diff
@@ -37,8 +38,37 @@ MAP                                                                        | BM2
 [TREC 2011 Microblog Track](http://trec.nist.gov/data/microblog2011.html)  | 0.3448 | 0.3719 | 0.3632 | 0.4112
 [TREC 2012 Microblog Track](http://trec.nist.gov/data/microblog2012.html)  | 0.2064 | 0.2230 | 0.2203 | 0.2541
 
-P30                                                                        | BM25   |BM25+RM3| QL     | QL+RM3
+P@30                                                                        | BM25   |BM25+RM3| QL     | QL+RM3
 ---------------------------------------------------------------------------|--------|--------|--------|--------
 [TREC 2011 Microblog Track](http://trec.nist.gov/data/microblog2011.html)  | 0.3741 | 0.4034 | 0.3905 | 0.4306
 [TREC 2012 Microblog Track](http://trec.nist.gov/data/microblog2012.html)  | 0.3237 | 0.3305 | 0.3249 | 0.3475
 
+### Tweets2013 collection
+Indexing the Tweets2013 collection:
+
+```
+nohup sh target/appassembler/bin/IndexCollection -collection TwitterCollection  -input \
+/path/to/Tweets2013/ -generator JsoupGenerator  -index lucene-index.Tweets2013.pos+docvectors -threads 32 \
+-storePositions -storeDocvectors -optimize > log.Tweets2013.txt &
+```
+
+Running topics from TREC 2013 (also look in `src/main/resources/topics-and-qrels/` for topics from TREC 2014):
+
+```
+sh target/appassembler/bin/SearchTweets -collection Twitter -index lucene-index.Tweets2013.pos+docvectors -bm25 \
+-hits 1000 -topics src/main/resources/topics-and-qrels/topics.microblog2011.txt -output run.mb13.bm25.txt
```

Comment:
The 2011 topics on the 2013 collection? Also I'd suggest that the indexing command differences between 2011/2013 are so minor that repeating the commands isn't necessary (much like you did with retrieval and `trec_eval` commands).